Release of first version of random generator policy

steveyuwono opened the pull request:
<!--- Please Provide general Summary on the PR Title -->

## Description
<!--- Describe in detail what is in this PR -->

## Background
<!--- What is the reason behind this PR -->
<!--- If this relates to a open issue, please provide the link issue -->
<!--- Please write "closes #123" to automatically close issue #123 on merge -->

## Checklists:
<!--- Please go over the checklist, and make sure you follow the guidelines -->
<!--- If you have any question, please ask -->
- [ ] Read the [CONTRIBUTION](https://github.com/fhswf/MLPro/blob/master/CONTRIBUTING.md) guide (**required**).
- [ ] Update the history on the source code (**required**).

detlefarend reviewed (changes requested):
Hi Steve, I just took a look and I have three annotations. First, an own constructor for the RandomGenerator class is not neccessary because seeding is optional. Class Training explicitely seeds the model of a scenario. And the inherited Agent class seeds the internal policy. The second point regards to the (non-)existence of boundaries. I am not sure whether it is better to raise an exception if boundaries are not set. Maybe we should set +- infinite as boundaries if they are not provided. Something to discuss...Third, the howtos should be changed in the current documentation branch. Otherwise we potentially get collisions. For instance, I have to eliminate all blanks from the filenames of all howtos (#327). I've not yet finished this, so it's not yet a problem.

steveyuwono commented:
Hi Detlef, thank you for the review. First, I have reverted the changes on howto files and will do these changes in the documentation branch. Second, I have removed the seeding and constructor from the class. Third, regarding the undefined base set or boundaries, I think that it makes sense to raise an exception if the base set or boundaries are not defined. But we can discuss this in the meeting.
